tasks/run-matlab-build/v0/task.json

```diff
@@ -0,0 +1,34 @@
+{
+    "": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "c61c6fcc-00f1-4aa0-8513-31aad3340512",
+    "name": "RunMATLABBuild",
+    "friendlyName": "Run MATLAB Build",
+    "description": "Run builds using the MATLAB build tool. Use this command to run the tasks in the plan returned by a file named buildfile.m in the root of your repository. To use the run-build command, you need MATLAB R2022b or a later release.",
```

Comment:
Should we use singular instead of plural here?

"Run a build using the MATLAB build tool."

Reply:
1) I agree with Mark. "a build" is more accurate.
2) command  --> task (x2)
3) run the software-build tasks specified in a file named buildfile.m.

Reply:
fixed